### DISCUSSION TITLE
About the ideas to be generated

### DISCUSSION THREAD

**Discussion by ErnestCodes** [hellocontento/full-stack-assignment-202410#3] [Q&A] [answered]
Are the ideas to be generated at random or are we to integrate an AI tool for this?

**Comment by knvrstrpn** [comment, marked as answer]
Integrate an AI tool.
The user needs to have real brainstorm sessions with a real AI.
In the chat interface, the user will ask the AI to save a certain idea they came up with together, and then the idea will be shown in the ideas list.
